docs-src/docs/dataset-exploration/uploading-predictions.md

```diff
@@ -56,6 +58,23 @@ except requests.exceptions.RequestException as e:
     print(f"Request error: {e}. {msg}")
 ```
 
+Predictions groups connected to segmentation datasets require one extra parameter called `classMapping`. This is used to map the class names 
+in the predictions to the class names in the annotations. The `classMapping` parameter is a list of dictionaries, where each dictionary
+contains the keys `annotated` and `predicted`. The `annotated` key is the class name in the annotations, and the `predicted` key is the 
+class name in the predictions. All class names in the predictions and the annotations must be present in the class mappings, even if they 
+don't need to be mapped.
+
```

Comment:
Maybe we could expand the example a little by also adding
`{"annotated": "oak", "predicted": "tree"},`
and a short description like "if you have annotated different species of trees, but predicts only whther it is a tree or not".